contributing.md

```diff
@@ -421,9 +424,7 @@ Porting changes across branches:
         # Fix conflicts
         git commit -sam"Backport from N to M"
         # Open pull request
-
-* When merging a port pull request, **do not squash or rebase**, create a merge
-  commit instead.
+        # Do not squash, rebase
```

Comment:
With the comma between squash and rebase, I found this sentence to be a little confusing - does it mean "do not squash, rebase instead" or does it mean "do not squash or rebase"? Perhaps we can modify this to make it clearer.

Reply:
I actually meant "do not squash, rebase instead", I'll update

Reply:
824176b170a8353a14f69bde100af7982fd8aa53